x-pack/plugins/uptime/public/components/monitor/synthetics/waterfall/components/styles.ts

```diff
@@ -83,9 +83,9 @@ interface WaterfallChartSidebarContainer {
 }
 
 export const WaterfallChartSidebarWrapper = euiStyled(EuiFlexItem)`
-  max-width: ${SIDEBAR_GROW_SIZE * 10}%;
   z-index: ${(props) => props.theme.eui.euiZLevel5};
-`;
+  min-width: 0;
```

Comment:
My original max-width solution was kinda an antipattern. This is the appropriate way to handle flexbox and no-wrap children.